### PR TITLE
Add the creation of default LoggingConfig, if config-logging is missing

### PR DESCRIPTION
**Pull request by houshengbo**

### CODE REVIEW

**Comment by greghaynes** [comment]
I wonder if we want to make this opt-in to preserve existing behavior by default for consumers? e.g. for serving we may want to preserve the existing behavior since it bubbles up potential errors in configuration rather than silently switching to default behavior.

**Comment by yanweiguo** [comment]
Current behavior is:
1. All required ConfigMaps, included `config-logging` must exist. Their data can be empty.
1. Any empty field will be assigned a default value.

This PR changes to creating an empty one if `config-logging` ConfigMap does not exist. Agree with @greghaynes, it's worth discussing whether we want to this change for missing ConfigMap.

/assign @mattmoor

**Comment by mattmoor** [comment]
/hold
I'd kinda like to understand the motivation for this better before it goes in.

File: injection/sharedmain/main.go [hunk 1]
```diff
@@ -77,7 +78,11 @@ func GetConfig(masterURL, kubeconfig string) (*rest.Config, error) {
 func GetLoggingConfig(ctx context.Context) (*logging.Config, error) {
 	loggingConfigMap, err := kubeclient.Get(ctx).CoreV1().ConfigMaps(system.Namespace()).Get(logging.ConfigMapName(), metav1.GetOptions{})
 	if err != nil {
-		return nil, err
+		if apierrors.IsNotFound(err) {
+			return logging.NewDefaultConfig(), nil
```
[the remark this answers is not in the document]
Comment (houshengbo) [reply injection/sharedmain/main.go]: Thx  done

**Pull request merged**